.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+## Expected behavior
+
+## Actual behavior
```

Comment:
We could use this format for it this step:

- When I perform a backup
- it throw the following error: